### DISCUSSION TITLE
logo图咋裂了🤣

### DISCUSSION THREAD

**Discussion by BiFangKNT** [lobehub/lobehub#3908] [Self Hosting] [answered]
如图🤣
![image](https://github.com/user-attachments/assets/bb8c2044-f4a7-4b4c-b37a-cc18271dc15f)

**Comment by BiFangKNT** [comment]
设置-关于里，以及欢迎页的logo也是，重启也不管用

**Reply by arvinxx** [reply]
你是不是用了 `.top` 域名

**Reply by BiFangKNT** [reply]
.top域名不能用吗？我记得没说这个域名不行啊

**Reply by arvinxx** [reply, marked as answer]
https://github.com/lobehub/lobe-chat/discussions/2828

**Reply by BiFangKNT** [reply]
我应用安装的时候能正常显示啊，应用图标也是正常的，也许网络波动了？
![image](https://github.com/user-attachments/assets/e45bdf6d-f8e5-4683-a8f9-082af1e19c63)
他说到的黑屏我这里没有，只是网页里的logo裂了

**Reply by BiFangKNT** [reply]
> #2828

我单独给我的域名做CDN加速，这个方案可行吗？

**Reply by BiFangKNT** [reply]
@arvinxx 要是域名被墙了的话，为什么只是logo显示不了呢？我在外部网络也能直连我的域名下的网页啊

**Reply by Matinal111** [reply]
没用，top 域名被 npmmirror 拉黑名单了，目前只能换域名解决

**Reply by arvinxx** [reply]
后面倒是可以给一个变量去切换npmmirror或者 unpkg

**Reply by BiFangKNT** [reply]
> 后面倒是可以给一个变量去切换npmmirror或者 unpkg

换unpkg的话有什么影响吗？何时安排？😳

**Reply by arvinxx** [reply]
@BiFangKNT unpkg是全球的npm包加速cdn，但国内可能被墙了

**Reply by BiFangKNT** [reply]
> @BiFangKNT unpkg是全球的npm包加速cdn，但国内可能被墙了

试过阿里云和腾讯云的cdn吗？

**Reply by BiFangKNT** [reply]
> @BiFangKNT unpkg是全球的npm包加速cdn，但国内可能被墙了

这些有试过吗
![image](https://github.com/user-attachments/assets/29cee5a9-3a86-48fa-a7f7-9a085afbf65c)
[博客链接](https://www.cnblogs.com/LiangSenCheng/p/17325563.html)
